systemd/suse-migration-grub-setup.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Update grub to migrated version
+
```

Comment:
We are missing the dependencies here. Should add

```
After=suse-migration.service
Requires=suse-migration.service
```

Reply:
Also the comment I think could be more specific, saying that we recreate the grub configuration file